Microsoft.Toolkit.Uwp.SampleApp/Microsoft.Toolkit.Uwp.SampleApp.csproj

```diff
@@ -125,7 +125,7 @@
       <Version>2.0.1</Version>
     </PackageReference>
     <PackageReference Include="Newtonsoft.Json">
-      <Version>10.0.3</Version>
+      <Version>12.0.3</Version>
```

Comment:
@ratishphilip are you using JSON.NET we're removing the dependency to use `System.Text.Json` instead, see PR #3414.

Reply:
@michael-hawker The Win2d Path Mini Language Parser code is not using JSON.NET.
Visual studio must have updated this package reference.
How do I remove it from the PR?